multi-repo-action/go.mod

```diff
@@ -0,0 +1,17 @@
+module cli
```

Comment:
Could the module name be something like this  `github.com/ossf/scorecard-actions/cli`?

https://github.com/ossf/scorecard/blob/6930c3ab3b715808a0d28d383c73b151bb495fe2/tools/go.mod#L1